File: 1009.c
/*
  Autor: Mateus Freitas;
  Nome: Salário com Bônus;
*/

#include <stdio.h>
 
int main() {
    char vendedor;
    double salarioFixo, vendas, total;
    
    scanf("%s", &vendedor);
    scanf("%lf%lf", &salarioFixo, &vendas);
    total = salarioFixo + (0.15*vendas);
    printf("TOTAL = R$ %.2lf\n", total);
    
 
    return 0;
}

File: Soluções Python/1009.py
#Autor: Mateus Freitas
#Nome: Salário com Bônus

nome = str(input())
salarioFixo = float(input())
vendas = float(input())

total = salarioFixo + (vendas * 0.15)

print ('TOTAL = R$ %0.2f' %total)
